packages/components/src/components/Dropdown/Dropdown.tsx

```diff
@@ -87,7 +87,9 @@ export const Dropdown = forwardRef(
             isDisabled,
             renderOnClickPosition,
             addon,
-            alignMenu = 'left',
+            alignMenu = 'bottom-left',
+            offsetX,
```

Comment:
You can customize position of the dropdown. Example here:

<img width="848" alt="Screenshot 2023-12-20 at 14 12 36" src="https://github.com/trezor/trezor-suite/assets/858321/5d652011-d3a4-4ff4-b52e-6ff87ef7b413">

In the case of notification dropdown:

<img width="724" alt="image" src="https://github.com/trezor/trezor-suite/assets/858321/c2ea2483-e3b0-445d-b2c7-e11c488a950e">

<img width="716" alt="image" src="https://github.com/trezor/trezor-suite/assets/858321/f98ab7e4-c552-474e-99ae-8bd47e852146">

Reply:
Not sure it's needed since i believe it should be unified somehow. Why do you think extra effset should be applied in this case? I am afraid these props will be abused later.

Reply:
Maybe it's my subjective opinion and ADHD, but it looks a bit like optical illusion when dropdown background is white and button is gray. It looks like these shapes are not in one line. I think we should correct it to fake it (~ -2px or move it more to show clear intention).

<img width="752" alt="image" src="https://github.com/trezor/trezor-suite/assets/858321/4bb77738-afa9-430a-ae5a-3ef6eb6292bc">

In general I think it's pretty common use case correct these positions in dropdown. For me it's a bit similar like using override through styled-components. You want to add some padding so you are able to do it, buw why not here?

Reply:
Yeah, I agree that it looks like an illusion :D I immediately want to come up with an algorithm and ratios to automate those offsets, so that they are relative to the toggle size or the dropdown size 😅  Because with the original toggle it does have an offset but it's less noticeable with a larget toggle, a `NavigationItem` in this case.

But yeah, leave the props! 🙌